dmcontent/questions.py

```diff
@@ -606,7 +652,7 @@ def value(self):
 class ListSummary(QuestionSummary, List):
     @property
     def value(self):
-        # Look up display values for options that have different labels from values
+        # TODO look up display values for options that have different labels from values
```

Comment:
So the comment was copy-pasted way-back-when, but actually was wrong. So I'm correcting the comment (but in fact @samuelhwilliams has a PR for this on the way :) )